src/views/certifications/content/schemas/landing-page.ts

```diff
@@ -11,6 +11,10 @@ export const LandingPageSchema = z.object({
 		heading: z.string(),
 		description: z.string(),
 	}),
+	/**
+	 * Note: these must be valid ProgramSlug values.
```

Comment:
Should we define the strings explicitly here?

Reply:
Like in the schema, im not familiar with zod but I remember doing this with joi

Reply:
Ah yeah great call - I've updated to use `zod.enum` here, and am also now inferring the `ProgramSlug` from that enum rather than re-declaring it ✨ great call!